0-bootstrap/README.md

```diff
@@ -342,11 +351,160 @@ The following steps introduce the steps to deploy with Cloud Build Alternatively
+   git clone https://github.com/terraform-google-modules/terraform-example-foundation.git
+
+   mkdir gcp-bootstrap
+
+   cp -R terraform-example-foundation/0-bootstrap/* gcp-bootstrap/
+
+   cp terraform-example-foundation/.gitignore gcp-bootstrap
+   ```
+
+1. Navigate to `gcp-bootstrap` and initialize a local Git repository to manage versions locally. Then, Create the environment branches.
+
+   ```bash
+   cd gcp-bootstrap
+
+   git init
+   git checkout -b plan
+
+   git checkout -b shared
+   ```
+
+1. Rename `terraform.example.tfvars` to `terraform.tfvars` and update the file with values from your environment:
+
+   ```bash
+   mv terraform.example.tfvars terraform.tfvars
+   ```
+
+1. Rename `cb.tf` to `cb.tf.example`:
+
+   ```bash
+   mv cb.tf cb.tf.example
+   ```
+
+1. Comment Cloud Build related outputs at `outputs.tf`.
+
+1. In `sa.tf` file, comment out lines related to Cloud Build. Specifically, search for `cicd_project_iam_member` and comment out the corresponding module, as well as the "depends_on" meta-argument in any modules that depend on the commented module.
+
+1. Use the helper script [validate-requirements.sh](../scripts/validate-requirements.sh) to validate your environment:
+
+   ```bash
+   ../terraform-example-foundation/scripts/validate-requirements.sh -o <ORGANIZATION_ID> -b <BILLING_ACCOUNT_ID> -u <END_USER_EMAIL>
+   ```
+
+   **Note:** The script is not able to validate if the user is in a Cloud Identity or Google Workspace group with the required roles.
+
+1. Run `terraform init` and `terraform plan` and review the output.
+
+   ```bash
+   git checkout plan
+   terraform init
+   terraform plan -input=false -out bootstrap.tfplan
+   ```
+
+1. Create a new folder called gcp-policies at the same directory level as the `terraform-example-foundation` folder. Initialize a Git repository, create a branch called `main`, and copy the contents of the `policy-library` directory from the `terraform-example-foundation` folder into the gcp-policies folder.
+
+   ```bash
+   cd ../
+
+   mkdir gcp-policies
+
+   cd gcp-policies
+   git init
+   git checkout -b main
+   cp -RT ../terraform-example-foundation/policy-library/ .
+   ```
+
+1. Commit changes to the main branch of the policy repo. This way you can manage versions locally.
+
+   ```bash
+   git add .
+   git commit -m 'Initialize policy library repo'
+   ```
+
+1. Navigate out of the policies repo.
+
+   ```bash
+   cd ..
+   ```
+
+1. To  validate your policies, run `gcloud beta terraform vet`. For installation instructions, see [Install Google Cloud CLI](https://cloud.google.com/docs/terraform/policy-validation/validate-policies#install).
+
+1. Run the following commands and check for violations:
+
+   ```bash
+   export VET_PROJECT_ID=pjr-seed-serverless-test
+   terraform show -json bootstrap.tfplan > bootstrap.json
+   gcloud beta terraform vet bootstrap.json --policy-library="../terraform-example-foundation/policy-library" --project ${VET_PROJECT_ID}
```

Comment:
I believe the correct would be:
`gcloud beta terraform vet bootstrap.json --policy-library="./terraform-example-foundation/policy-library" --project ${VET_PROJECT_ID}`

Reply:
ACK - i'm making this adjustment in another branch and will make a PR to your branch @amandakarina